sw_watchdog_msgs/CMakeLists.txt

```diff
@@ -21,16 +21,10 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   LIBRARY_NAME ${PROJECT_NAME}
 )
 
-# if(BUILD_TESTING)
-#   find_package(ament_lint_auto REQUIRED)
-#   ament_lint_auto_find_test_dependencies()
-
-#   find_package(ros_testing REQUIRED)
-#   add_ros_test(
-#     test/test_watchdog.py
-#     TIMEOUT 60
-#   )
-# endif()
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
```

Comment:
I don't think `ament_lint` can lint message files.

Reply:
You are probably right. I took the cmake configuration from `lifecycle_msgs`, though, as I thought being a quality level 1 package this probably knows what it's doing. :D